packages/preview/ansi-render-0.1.0/README.md

```diff
@@ -0,0 +1,59 @@
+# ANSI Escape Sequence Renderer
+
+This script provides a simple way to render text with ANSI escape sequences. It uses the `ansi_render` function from the `ansi_render.typ` module to render text with various formatting options, including font, font size and theme.
+
+contribution is welcomed!
+
+## Usage
+
+```typst
+#import "@preview/ansi_render:0.1.0": *
```

Comment:
```diff
- #import "@preview/ansi_render:0.1.0": *
+ #import "@preview/ansi-render:0.1.0": *
```

Note the `-` vs `_`, took me longer than it should have to realize why the import wasn't working